waffle_configs/bedrock-prod.env

```diff
@@ -13,44 +13,26 @@ FUNNELCAKE_139_LOCALES=en-US
 FUNNELCAKE_139_PLATFORMS=win
 
 # switches
-SWITCH_BETA_WHATSNEW_68_TRAILHEAD=on
 SWITCH_CONTENTFUL_HOMEPAGE_DE=off
 SWITCH_CONTENTFUL_HOMEPAGE_EN=on
-SWITCH_CONTRIBUTE_SURVEY=off
 SWITCH_CONVERT_EXPERIMENT_SCRIPT=on
-SWITCH_DATA_PRIVACY_DAY_2017=on
-SWITCH_DEV_WHATSNEW_68=on
-SWITCH_DEV_WHATSNEW_68_TRAILHEAD=on
 SWITCH_DISABLE_SHA1_DOWNLOADS=off
-SWITCH_ELECTION_BUNDLE=on
```

Comment:
Election stuff was removed in https://github.com/mozilla/bedrock/commit/8b2b3bf262f72efc12efbb59405b7f76cc675f81